releasenotes/notes/aml-add-file-limit-info-6ee9898dade59359.yaml

```diff
@@ -0,0 +1,17 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    In agent.log, a DEBUG, WARN, and ERROR log have been added to report
+    how many file handles the core Agent process has open. The DEBUG log
+    reports the info, the WARN log appears when the core Agent is over 90%
+    of the OS file limit, and the ERROR log appears when the core Agent
+    has reached 100% of the OS file limit. In status.log, fields 
```

Comment:
Not very important but I think `status.log` refers to the file that contains the `status` output in flares. 
Since it's the general status output, "status command" would be more accurate imho (not a strong opinion so feel free to ignore)